content/articles/create-screen-recorder-in-javascript/index.md

```diff
@@ -0,0 +1,219 @@
+title: Section Engineering Education
+
+description: Resources created by engineers for engineers
+
+aliases:
+  - '/create-screen-recoder/'
+  - '/javascript-screen-recorder/'
+type: articles
+
+images:
+  - url: /engineering-education/create-screen-recorder-in-javascript/hero.jpg
+  
+    alt: Screen Recorder in JavaScript
+
+
+### Introduction
+The Screen Capture API is used to capture the screen or a part of the screen as a Media Stream. From the recorded Media Stream we can create a video file.
+
+### Prerequisites
+
+To follow along with this tutorial, you should have:
+
+-   A basic understanding of HTML and JavaScript.
+
+-   A code editor. I’ll be using  [Visual Studio Code](https://code.visualstudio.com/download).
+
+-   A browser to view the webpage, preferably  [Google Chrome](https://www.google.com/intl/en_in/chrome/).
+
+### Project directory
+
+Create a new directory for the project and create   `index.html`  and  `recordscreen.js` files.
+
+```bash
+screen_recorder_project/
+|-index.html
+|-recordscreen.js
+```
+
+### HTML Page
+On the `index.html` page there will be
+* Two buttons one to Start Screen Recording another to Stop Screen Recording.
+* One `div` to display the recording status, the style for the div is also added in the `index.html` file.
+
+```html
```

Comment:
Why not have a CSS sheet for styling rather than mixing it up with the index.html?
I think it is the most advisable way of doing this.

Reply:
i have changed this